Simplified/Book/Models/NYPLBookRegistry.h

```diff
@@ -259,3 +266,28 @@ genericBookmarks:(nullable NSArray<NYPLBookLocation *> *)genericBookmarks;
                       block:(void (^_Nonnull)(void))block;
 
 @end
+
+#if FEATURE_AUDIOBOOKS
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Weverything"
```

Comment:
The compiler shows like 60+ warnings because the protocol is defined in Swift and/or NYPLAudiobookBookmark class is a forward declaration in this header file. I am adding these lines to muted the warnings.

Reply:
Fixed, thanks for catching that.